packer-build/test-ubuntu1804.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+sudo shutdown 1
```

Comment:
Does this mean "one minute" or "one second"? Or at epoch "1"? Should there be a `+`? Anyway, it feels safer to move it lower, as the last step. Or better yet, just kill the VM with fire after the test is done.

Reply:
It's one minute. I don't know any other way of stopping the machine without implicitly killing the ssh connection (that would result in a non-zero code for the ssh command in `.drone.yml`).

Reply:
Yeah, it's easier if you kill the qemu vm when you're done.